garbagetruck/garbagetruck.go

```diff
@@ -0,0 +1,59 @@
+package garbagetruck
+
+import (
+	"github.com/ian-kent/go-log/logger"
```

Comment:
Good catch - we need to collect all the licenses & ensure we meet them.
Will come up with a solution and circle back.

On Fri, Jul 29, 2016 at 3:32 PM, Khem Raj <notifications@github.com> wrote:

> In garbagetruck/garbagetruck.go
> <https://github.com/Comcast/webpa-common/pull/13#discussion_r72870178>:
>
> > @@ -0,0 +1,59 @@
> > +package garbagetruck
> > +
> > +import (
> > +	"github.com/ian-kent/go-log/logger"
>
> this is MIT licensed external component. Where do we represent this
> information when we build webpa ?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Comcast/webpa-common/pull/13/files/c6a32bf3177f368e692ad43e7a279972f90edae7#r72870178>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAqpyQMnkShpsRunrAUG9JVbSc8eeBa3ks5qan90gaJpZM4JYoN7>
> .
>